docs/pages/sidenavigation.js

```diff
@@ -0,0 +1,638 @@
+  generatedDocGen,
+}: {|
+  generatedDocGen: {| [string]: DocGen |},
+|}): Node {
+  return (
+    <Page title={generatedDocGen.SideNavigation?.displayName ?? ''}>
+      <PageHeader
+        name={generatedDocGen.SideNavigation?.displayName}
+        description={generatedDocGen.SideNavigation?.description}
+        shadedCodeExample
+        defaultCode={`
+<SideNavigation accessibilityLabel="Main example">
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="bell"
+    item={{ label: 'Notifications', value: 'notifications' }}
+    counter={{ number: '20', accessibilityLabel: 'You have 20 notifications in your inbox' }}
+    notificationAccessibilityLabel="New notifications"
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="speech"
+    item={{ label: 'Messages', value: 'messages' }}
+    counter={{ number: '10', accessibilityLabel: 'You have 10 messages in your inbox' }}
+    notificationAccessibilityLabel="New messages"
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="cog"
+    item={{ label: 'Settings', value: 'settings' }}
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="lock"
+    item={{ label: 'Business Access', value: 'business' }}
+  />
+  <SideNavigation.Item
+    href="#"
+    onSelect={({ event }) => event.preventDefault}
+    icon="add-layout"
+    item={{ label: 'Tune your home feed', value: 'tune_homefeed' }}
+    badge={{ text: 'New', type: 'info' }}
+  />
+</SideNavigation>
+        `}
+        slimBanner={
+          <SlimBanner
+            type="warning"
+            iconAccessibilityLabel="Warning"
+            message="This component is on alpha phase, still under developoment. The component will support three nested levels and accessibility keyboard navigation. The component will change behavior and the API might also change in future component version releases."
+          />
+        }
+      />
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen.SideNavigation} />
+
+      <AccessibilitySection name={generatedDocGen.SideNavigation?.displayName}>
+        <MainSection.Subsection
+          title="Active item"
+          description='Sidenavigation.Item has an "active" state that visually identifies it. To set Sidenavigation.Item to active pass "page" (page redirect) or "section" (selected content section). Use routing hooks from React.Router or other frameworks to identify the current route. For example, if the current pathname matches the Sidenavigation.Item href, set Sidenavigation.Item to "page".'
+        >
+          <MainSection.Card
+            shaded
+            defaultCode={`
+function Example() {
+  const reactRouterPath = '/sidenavigation'
+
+  return (
+    <Box height={300} overflow="scroll">
+      <SideNavigation accessibilityLabel="Main example">
```

Comment:
I think your test failure will be fixed if you change this label (duplicated from top example)

Reply:
DONE